dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/tei/query/TeiFullQueryTest.java

```diff
@@ -78,7 +77,7 @@ void testSqlQueryRenderingWithOrgUnitNameObject()
         // then
         assertTrue( sql.contains( "ouname" ) );
 
-        assertContains( "order by t_1.\"OUNAME\" ASC", sql );
+        assertContains( "order by t_1.\"OUNAME\" ASC".toLowerCase(), sql.toLowerCase() );
```

Comment:
Should we remove the "toLowerCase()", and match the correct string/case expected?

Reply:
I do not think so, it is just about if the string fraction is there and there is nothing case sensitive